lib/chef-acceptance/cli.rb

```diff
@@ -63,25 +58,86 @@ def test(test_suite)
       end
     end
 
-    no_commands do
-      def test_suite_exist!(test_suite)
-        unless File.exist?(test_suite)
-          abort <<-EOS
-Could not find test suite '#{test_suite}' in the current working directory '#{Dir.pwd}'.
-Make sure to have a test suite with a '#{File.join(ACCEPTANCE_DIR, ACCEPTANCE_COOKBOOK_NAME)}' directory.
-EOS
-        end
+    desc 'generate', 'Generate acceptance scaffolding'
+    def generate(name)
+      suite = TestSuite.new(name)
+
+      abort "Test suite '#{name}' already exists." if suite.exist?
+
+      [suite.acceptance_cookbook_dir, suite.chef_dir, suite.recipes_dir].each do |path|
+        FileUtils.mkpath path
+      end
+
+      CORE_ACCEPTANCE_COMMANDS.each do |file|
+        FileUtils.touch(File.join(suite.recipes_dir, "#{file}.rb"))
       end
 
-      def executable_installed!(executable)
+      File.write(File.join(suite.acceptance_cookbook_dir, 'metadata.rb'),
+        "name '#{ACCEPTANCE_COOKBOOK_NAME}'")
+
+      File.write(suite.chef_config_file,
+        "cookbook_path File.join(File.dirname(__FILE__), '..', '..')")
+
+      File.write(File.join(name, '.gitignore'),
+        "local-mode-cache/\nnodes/")
+
+      puts "Run `chef-acceptance test #{name}`"
+    end
+
+    desc 'version', 'Print chef-acceptance version'
+    def version
+      puts ChefAcceptance::VERSION
+    end
+
+    no_commands do
+      def executable_installed?(executable)
         exts = ENV['PATHEXT'] ? ENV['PATHEXT'].split(';') : ['']
         ENV['PATH'].split(File::PATH_SEPARATOR).each do |path|
           exts.each do |ext|
             exe = File.join(path, "#{executable}#{ext}")
-            return exe if File.executable?(exe) && !File.directory?(exe)
+            return true if File.executable?(exe) && !File.directory?(exe)
+          end
+        end
+        false
+      end
+
+      def chef_client_installed!
+        unless executable_installed? 'chef-client'
+          abort 'chef-client not installed'
```

Comment:
Yes.  Cross platform without relying on any external tool (like `which` or `where`).  I'll update the message. I was fuzzy on it.